ios/sdk/WeexSDK/Sources/Component/WXScrollerComponent.m

```diff
@@ -35,6 +35,35 @@ @interface WXScrollerComponnetView:UIScrollView
 @implementation WXScrollerComponnetView
 @end
 
+@interface UIScrollView (Events)
```

Comment:
I think that the name of Events category for UIScrollview  is not long enough avoiding conflict later.